coralproject/talk
# Added support for deep comment counts

## What does this PR do?

Adds support for deep comment counts to be returned. It does this by overriding the `replyCount` edge to provide the new loader to use instead.

**Warning: This feature introduces possibly a large performance impact for sites with many levels of nesting. Not recommend for any site exceeding 3 levels deep of commenting.**

Related: https://www.pivotaltracker.com/story/show/150848457

## How do I test this PR?

1. Add `talk-plugin-deep-reply-count` to the `plugins.json` file.
2. Reload server.
3. Notice that counts returned are deep!
